charts/charts/backup-operator/values.yaml

```diff
@@ -27,13 +27,21 @@ securityContext: {}
   # runAsNonRoot: true
   # runAsUser: 1000
 
-resources:
+prod_resources:
   limits:
-    cpu: 4
-    memory: 8G
-  requests:
     cpu: 2
     memory: 4G
+  requests:
+    cpu: 500m
+    memory: 512Mi
+
+test_resources:
```

Comment:
the convention in Go/Yaml is to use mixedCaps rather than underscores to write multiword names.  what do you think ?

Reply:
done